model/data.py

```diff
@@ -19,7 +19,7 @@ class Config:
     commit: Optional[str] = Field(
         description="Commit of the model. May be empty if not yet committed."
     )
-    hash: str = Field(description="Hash of the trained model.")
+    hash: Optional[str] = Field(description="Hash of the trained model.")
```

Comment:
Can you please add a large comment about when we expect commit and hash to be empty vs. when it should be populated?